data/ui/sample_box.blp

```diff
@@ -21,6 +21,15 @@ template SampleBox : Box {
       hexpand: true;
     }
 
+    Button color_button {
+      tooltip-text: _("Pick a color");
+      styles ["flat"]
+      Image {
+        hexpand: false;
+        icon-name: "color-picker-symbolic";
```

Comment:
Add the following to restore the slightly larger size of the color picker:

        pixel-size: 20;

Reply:
Otherwise this looks good, a bit cleaner solution like this.